jekyll/_cci2/vm-service.adoc

```diff
@@ -66,6 +68,18 @@ You will need to associate the `circleci` user with the image you want to use as
 
 To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`).
 
+=== Creating a Windows AMI
+_Introduced in CircleCI Server v2.18.3_
+
+Creating a Windows image and specifying it under the VM Service settings lets your users run jobs on dedicated Windows VMs. To create your Windows image run through the following steps. Windows images are built on CircleCI, so we suggest you run through this process once your installation is up and running. Alternatively you can use any other CircleCI account – including on our managed Cloud service – to generate the image:
+
+. Create a new repository under your GitHub or GitHub Enterprise account.
+. Copy the contents of **<INSERT WINDOWS IMAGE BUILDER REPO LINK>** into your new repo.
+. Open up your installation of CircleCI Server and https://circleci.com/docs/2.0/getting-started/#setting-up-your-build-on-circleci[connect your new repo] by clicking Follow Project from the Add Projects page.
+. Click Start Building to run the Windows image buider.
```

Comment:
This won’t work actually — at this point the AWS keys aren’t set up. I think there isn’t a way other than "click Start Building, go ahead and finish setting up the repo as indicated in the Example repo, and once you have the AWS keys set up, cancel the build and click Rebuild, or something like that.

Reply:
Ah yes, good point, I forgot about setting up the env vars - will add in

Reply:
Can they not set up their AWS keys as per the instructions in the example repo before adding the project to CircleCI from the UI? That way they don't have to start and stop building? - I'll make changes to reflect this now and we can see if it works I guess?